### PR TITLE
Qdb improvements

### PR DESCRIPTION
**Pull request by ucgJhe**
1. add basic support for x86
2. fix minor bugs

no assembly listing before current address for now
demo

### CODE REVIEW

**Comment by xwings** [comment]
hi,

already PR to rootfs ?

File: qiling/debugger/qdb/utils.py [hunk 1]
```diff
@@ -121,6 +131,17 @@ def get_cpsr(bits: int) -> (bool, bool, bool, bool):
             )
 
 
+def get_x86_eflags(bits: int) -> (bool, bool, bool, bool, bool, bool):
```
Comment (elicn) [inline qiling/debugger/qdb/utils.py:134]: This is not valid annotation. You may want to use `Dict[str, bool]` instead.

**Comment by elicn** [comment]
Adding an architectures is a good opportunity to think of a way to generalize the API and consider implementing this as an OOP. That could make additional architecture support a lot easier and less messy..

**Comment by ucgJhe** [comment]
> hi,
> 
> already PR to rootfs ?

it's an accident, fixed by following commit.

File: qiling/debugger/qdb/utils.py (hunk 1 above)
Comment (ucgJhe) [reply qiling/debugger/qdb/utils.py]: thx for the hint, fixed in following commit.

**Comment by xwings** [comment]
Happy New Year!

**Review by xwings — APPROVED** [review]

**Review by cla7aye15I4nd — APPROVED** [review]

**Pull request merged**